main.go

```diff
@@ -18,6 +20,101 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
+const (
+	diskPath             string = "./tmp/disk.img.gz"
+	diskPathDecompressed string = "./tmp/disk.img"
+	diskPathConverted    string = "./tmp/disk.qcow2"
+)
+
+func apply(vmName string) error {
```

Comment:
I would rename this function to something like `createVMExport`

Reply:
Done, I've changed it to `applyVMExport`.